app/models/user.rb

```diff
@@ -80,4 +80,8 @@ def requires_no_invitation_code!
   def requires_no_invitation_code?
     !!@requires_no_invitation_code
   end
+
+  def scenarios?
```

Comment:
Yes, that is more in line with the rest of the code base.

I try to avoid predicates that have "has" or "is" in front, but since that's the convention in Huginn I will adjust my coding style!